docs/ManualMocks.md

```diff
@@ -13,15 +13,15 @@ Manual mocks are used to stub out functionality with mock data. For example, ins
 Manual mocks are defined by writing a module in a `__mocks__/` subdirectory
 immediately adjacent to the module. For example, to mock a module called
 ``user`` in the ``models`` directory, create a file called ``user.js`` and
-put it in the ``models/__mocks__`` directory. If the module you are mocking is
-a node module (eg: `fs`), the mock should be placed in the same parent
-directory as the ``node_modules`` folder. Eg:
+put it in the ``models/__mocks__`` directory. If the module you are mocking 
+is a node module (eg: `fs`), the mock should be placed in the ``__mocks__`` 
+folder adjacent to ``node_modules``. Eg:
```

Comment:
Can we actually use single backticks and also make this paragraph sit in one line as previous one? Double backticks are unnecessary and I'm not sure why it was written this way. Let's fix it!